Cpu[] cpus
int32 memory_available
int32 memory_used
int32 num_processes

Filesystem[] filesystems
NetworkInterface[] network_interfaces

